teacher/take_attendance.html

```diff
@@ -360,9 +364,6 @@
 }
 </script>
 
-{% if not teacher %}
-Teacher not found! This error should never happen. Please inform the selectives team. Thanks!
-{% else %}
```

Comment:
Turns out, `teacher` is not used anywhere on these pages so there's no reason to pass it. Since now both admin and teachers can access this page, it doesn't make sense to check if the user is a teacher.